UDP/current_term_courses/v1_student_current_term_course.sql

```diff
@@ -6,7 +6,7 @@ WITH
  FROM
    `udp-umich-prod.context_store_entity.academic_term` at2
  WHERE
-   DATE(current_timestamp) < DATE_ADD(at2.term_end_date, INTERVAL 2 WEEK)
+   DATE(current_timestamp) <= DATE_ADD(at2.term_end_date, 10 DAY)
```

Comment:
Looks like the keyword 'INTERVAL' is missing from the statement:

DATE(current_timestamp) <= DATE_ADD(at2.term_end_date, INTERVAL 10 DAY)

Reply:
Thanks for catching that, I fixed and ran the full query as well this time